services/firebase/src/lib/digest.ts

```diff
@@ -0,0 +1,118 @@
+const { createSSRApp } = require("vue");
+const { renderToString } = require("vue/server-renderer");
+import mjml2html = require("mjml");
+import * as fs from "fs";
+import * as moment from "moment";
+import { firestore } from "../firebase";
+
+export async function renderEmail(type: string, data: any, customUtms = {}) {
+  const template = fs.readFileSync(`./templates/${type}.mjml`, "utf8");
```

Comment:
```suggestion
  const template = fs.readFileSync(`../templates/${type}.mjml`, "utf8");
```

wrong path to templates, here is how you fix it.